chapters/scoping.tex

```diff
@@ -327,15 +332,15 @@ \section{Simultaneous Inner/Outer Declarations}\doublelabel{simultaneous-inner-o
 may have modfications, those modifications are only applied to the
 \lstinline!inner! declaration.
 
-{[}\emph{Example}:
+\begin{example}
 \begin{lstlisting}[language=modelica]
 class A
   outer parameter Real p=2; // error, since modification
 end A;
 \end{lstlisting}
 
-\emph{Intent of the following example: Propagate} enabled \emph{through
-the hierarchy, and also be able to disable subsystems locally.}
+Intent of the following example: Propagate \lstinline!enabled! through
```

Comment:
I don't think this should be lstinline as is, since there is no variable called enabled (it is called isEnabled and enableMe). Using emph may work.

Reply:
Fixed.